src/DynamicFormControlGroup.vue

```diff
@@ -1,25 +1,25 @@
 <template>
-    <b-row class="my-2">
-        <label v-if="controlGroup.label" class="col-form-label col-md-5">{{controlGroup.label}}
+    <label class="row my-2">
+        <div v-if="controlGroup.label" class="col-form-label col-md-5">
```

Comment:
This `v-if` should be on the entire label, unfortunately - we definitely shouldn't have a label element if it's not going to have any text. Do we actually need this div at all? The label could have the col class and we could keep the existing row container. 

Also, it looks ok to have labels on non-input controls, but it's not really correct: https://stackoverflow.com/questions/33214093/is-it-good-practice-to-use-label-for-non-input-non-interactive-elements
So in that case the label should really be replaced with a span. (Sorry!)

Unfortunately, all this means that we'll have some repetition, pseudo-template would have to be:
```
<label v-if="controlGroup.label && !readonly">
  ..
  <dynamic-form-control/>
</label>
<div v-else>
  <span v-if="controlGroup.label && readonly>..</span>
  <dynamic-form-control/>
</div>
```
I can't think of any other way of doing it, because of the label nesting, can you?

Reply:
Yeah, I figured we'd have to repeat code if we wanted to make a special case for the read only stuff, I just didn't think it mattered, but I guess it does. I'll experiment to see what works.